migrations/006.do.user_email.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users ADD COLUMN email VARCHAR(255) NOT NULL;
```

Comment:
Ok, because it shows an exemple of legit use of migrations. But there is no production of the demo, so it is also ok to update existing migrations files.